pkg/hintrunner/zero/zerohint_math_test.go

```diff
@@ -10,6 +11,8 @@ import (
 )
 
 func TestZeroHintMath(t *testing.T) {
+	felt127 := new(fp.Element).SetBigInt(new(big.Int).Lsh(big.NewInt(1), 127))
```

Comment:
Why not use `&utils.Felt127`?

Reply:
I forgot to replace it after merging main haha